### PR TITLE
[docs]: Add transitions on expanding in Sidebar

### PR DESCRIPTION
**Pull request by Caladan08**
## Description

Title

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing guide](https://github.com/svelteuidev/svelteui/blob/main/CONTRIBUTING.md)
- [x] Prefix your PR title with `[@svelteui/core]`, `[@svelteui/actions]`, `[@svelteui/motion]`, `[@svelteui/core]`, `[core]`, or `[docs]`.
- [x] Include a description of the changes made in the PR description and in the commit messages.
- [ ] Include screenshots/videos of the changes made.
- [ ] Verify that the linter and tests are passing, with `yarn lint` and `yarn test` or just run `yarn prepush`.

### CODE REVIEW

**Comment by BeeMargarida** [comment]
I've been away from the computer this last week, will take a look at this tomorrow

File: apps/docs/src/routes/+layout.svelte [hunk 1]
```diff
@@ -47,42 +47,44 @@
...
 
 
-<PageTransition refresh={data.pathname}>
-  <div class="main" class:nosidebar={nosidebar}>
-		<article class="article" class:homepage={$page.url.pathname === '/'}>
-      <slot />
-    </article>
-  </div>
-</PageTransition>
+<div style="overflow: hidden">
```
Comment (BeeMargarida) [inline apps/docs/src/routes/+layout.svelte:50]: Nice catch :ok_hand:

**Review by BeeMargarida — COMMENTED** [review]
I notice one problem with this change: when navigating to another page, the sidebar animates when opened, which creates a weird effect:

https://github.com/svelteuidev/svelteui/assets/25725586/147c13cc-030d-4475-9f49-9f2ac7d4e93a

Perhaps it's best to remove the transitions in the sidebar sections for now.

**Comment by Caladan08** [comment]
> I notice one problem with this change: when navigating to another page, the sidebar animates when opened, which creates a weird effect:
> 
> Perhaps it's best to remove the transitions in the sidebar sections for now.

Yeah, for some reason, even though there is no change to the `sidebar` value when we navigate through the pages, it will still rerun the transition every time. I tried using `#key` as well but didn't work. Guess I gotta drop that idea

**Comment by BeeMargarida** [comment]
> Yeah, for some reason, even though there is no change to the `sidebar` value when we navigate through the pages, it will still rerun the transition every time. I tried using `#key` as well but didn't work. Guess I gotta drop that idea

Hum, did you try adding the `|local` modifier to the transitions?

**Comment by Caladan08** [comment]
> Hum, did you try adding the `|local` modifier to the transitions? Perhaps it fixes in this case

Yeah, that works!

**Comment by BeeMargarida** [comment]
> I've added the latest commit in my fork but there seems to be some issue with merging. Could you take a look at it?

It seems like a Github issue. It happened to me in another repo as well. Don't have the solution for it yet

**Comment by BeeMargarida** [comment]
Sorry, been busy, I'll pick it up today or tomorrow

**Comment by BeeMargarida** [comment]
Sorry again, I'll pick it up this week for sure!

**Comment by BeeMargarida** [comment]
Thank you for this contribution! 🙌 And so sorry for the delay, life got a bit hectic for a while, now back to normal.

**Pull request merged**